R/dataset-stream.R

```diff
@@ -0,0 +1,62 @@
+#' Get the pending streams for a dataset
+#'
+#' Crunch allows you to stream data to a dataset. Streaming data is useful for
+#' datasets which have frequent updates (see 
+#' [the Crunch documentation](http://docs.crunch.io/#streaming-rows) for more)
+#' information. `pendingMessages()` retrieves the number of pending messages and `appendStreamedRows()` appends all pending streamed rows to the dataset.
```

Comment:
Update the function names. And if you do `[functionName()]`, it will link to the man page for that function.